docs/release-info/release-notes.md

```diff
@@ -57,46 +57,460 @@ For tips about how to write a good release note, see [Release notes](https://git
+|`cgroup/disk/write/size`|Reports the number of bytes transferred from specific devices by a cgroup of process that is running.|`diskName`|Varies|
+|`cgroup/disk/read/count`|Reports the number of read operations performed on specific devices by a cgroup of process that is running.|`diskName`|Varies|
+|`cgroup/disk/write/count`|Reports the number of write operations performed on specific devices by a cgroup of process that is running.|`diskName`|Varies|
+
+[#16472](https://github.com/apache/druid/pull/16472)
+
+#### Default Prometheus metrics
+
+The default Prometheus metrics have been expanded to cover more of the available metrics for Druid.
+
+[#16340](https://github.com/apache/druid/pull/16340)
+
+#### Configurable connection timeouts
+
+Added HTTP client property `clientConnectTimeout` to enable configuration of connection timeouts for Druid HTTP client requests.
+
+[#16831](https://github.com/apache/druid/pull/16831)
+
+#### Added indexer metrics
+
+Added `indexer/task/failed/count` and `indexer/task/failed/count` metrics.
+
+[#16829](https://github.com/apache/druid/pull/16829)
+
+#### Added subquery metrics
+
+Added `subquery/rows` and `subquery/bytes` metrics, which indicate the size of the results materialized on the heap.
+
+[#16835](https://github.com/apache/druid/pull/16835)
+
+#### Added segment loading rate metrics
+
+Made the following changes to segment loading rate metrics:
+
+- Added `LoadingRateTracker` which computes a moving average load rate based on
+the last few GBs of successful segment loads.
+- Added `expectedLoadTimeMillis` to `/druid/coordinator/v1/loadQueue?simple` API response.
+- Updated the Coordinator to emit `segment/loading/rateKbps`.
+
+[#16691](https://github.com/apache/druid/pull/16691)
+
 ### Extensions
 
+#### Kubernetes improvements
+
+When the fabric client is unable to communicate with the Kubernetes server, the shutdown logic now proceeds without populating the task status location, ensuring all data structures are cleaned up.
+
+[#16711](https://github.com/apache/druid/pull/16711)
+
+#### `pjac4j` OIDC context
+
+The `pac4j` extension now returns the OIDC context for `AuthenticationResult`.
+
+[#16109](https://github.com/apache/druid/pull/16109)
+
+#### Middle Manager-less Druid improvements
+
+The Dynamic Pod Template Selection feature enhances the K8s extension by enabling more flexible and dynamic selection of pod templates based on task properties, which can optimize resource utilization and task execution efficiency.
+
+[#16510](https://github.com/apache/druid/pull/16510)
+
+In MiddleManager-less ingestion, Druid adds the pod template name as an annotation to the created job or pod.
+
+[#16772](https://github.com/apache/druid/pull/16772)
+
+#### Configure case sensitivity for Iceberg
+
+You can now use the `caseSensitive` Boolean config to configure how Druid reads column names from Iceberg.
```

Comment:
Perhaps mention that this is optional, and that the the Iceberg table scans has case sensitivity enabled by default.